overcast/runner/__init__.py

```diff
@@ -152,6 +152,19 @@ def __init__(self, name, info, runner, keypair=None, userdata=None):
         if self.info.get('flavor') in self.runner.mappings.get('flavors', {}):
             self.info['flavor'] = self.runner.mappings['flavors'][self.info['flavor']]
 
+        if self.info.get('boot_volume') in self.runner.mappings.get('volumes',{}):
+            self.info['boot_volume'] = self.runner.mappings['volumes'][self.info['boot_volume']]
+
+        ##
+        # This is to allow disallowing boot_volume, in case mapfile dont have
+        # any such settings and environment file have a reference name for
+        # boot_volume
+        ##
+        try:
+            just_tmp_value=int(self.info['boot_volume'])
```

Comment:
If you're simply trying to validate that `self.info['boot_volume']` is an integer (in a string), don't bother assigning it to a variable.